app/api/dao/mentorship_relation.py

```diff
@@ -60,9 +61,17 @@ def create_mentorship_relation(self, user_id, data):
         if not mentee_user.need_mentoring:
             return {'message': 'Mentee user is not available to be mentored.'}, 400
 
+        # check if both users(mentor and mentee) have verified their email
+        if mentee_user.is_email_verified:
```

Comment:
this if else block is non pythonic, if then pass else do something?

Reply:
@LordGameleo, read the comments carefully and see how pythonic code is written. I'm questioning this particular way you've done something, not saying that's how it is. if <cond> pass else <stmt> makes no sense at all.